.github/workflows/python-package-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Python Package Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  nox-tests:
+    runs-on: ubuntu-latest
+    container: ghcr.io/gridtools/gridtools-base:${{ matrix.compiler }}
+    strategy:
+        matrix:
+            python-version: ["3.10"]
```

Comment:
Shouldn't we test more python versions here (specially with such a broad range of supported versions in `setup.cfg`)? It should be fairly simple and tests shouldn't take long, right?

Reply:
With the recent changes, the wheel is now built and tested with 3.10 and then tested more Python versions.